res/css/views/rooms/_AppsDrawer.pcss

```diff
@@ -222,60 +222,43 @@ limitations under the License.
     }
 
     .mx_AppTileMenuBar_widgets {
-        float: right;
         display: flex;
-        flex-direction: row;
         align-items: center;
-    }
-
-    .mx_AppTileMenuBar_iconButton {
-        --size: 24px; /* Size of the button. Its height and width values should be same */
 
-        margin: 0 4px;
-        position: relative;
-        height: var(--size);
-        width: var(--size);
+        .mx_AppTileMenuBar_widgets_button {
+            --size: 24px; /* Size of the button. Its height and width values should be same */
 
-        &::before,
-        &:hover::after {
-            content: "";
-            position: absolute;
+            margin: 0 4px;
+            position: relative;
             height: var(--size);
             width: var(--size);
-        }
-
-        &::before {
-            background-color: $muted-fg-color;
-            mask-position: center;
-            mask-repeat: no-repeat;
-            mask-size: 12px;
-        }
-
-        &:hover::after {
-            background-color: $panel-actions;
-            border-radius: 50%;
-            left: 0;
-            top: 0;
-        }
-
-        &.mx_AppTileMenuBar_iconButton--collapse::before {
-            mask-image: url("$(res)/img/element-icons/minimise-collapse.svg");
-        }
-
-        &.mx_AppTileMenuBar_iconButton--maximise::before {
-            mask-image: url("$(res)/img/element-icons/maximise-expand.svg");
-        }
-
-        &.mx_AppTileMenuBar_iconButton--minimise::before {
-            mask-image: url("$(res)/img/element-icons/minus-button.svg");
-        }
-
-        &.mx_AppTileMenuBar_iconButton--popout::before {
-            mask-image: url("$(res)/img/feather-customised/widget/external-link.svg");
-        }
-
-        &.mx_AppTileMenuBar_iconButton--menu::before {
-            mask-image: url("$(res)/img/element-icons/room/ellipsis.svg");
+            display: flex;
+            align-items: center;
+            justify-content: center;
+
+            &:hover::after {
+                content: "";
+                position: absolute;
+                height: var(--size);
+                width: var(--size);
+                background-color: $panel-actions;
+                border-radius: 50%;
+                left: 0;
+                top: 0;
+            }
+
+            .mx_Icon {
+                > * {
+                    fill: $muted-fg-color;
+                }
+
+                &.mx_Icon--stroke {
+                    > * {
+                        fill: none; /* override the fill value */
+                        stroke: $muted-fg-color;
+                    }
+                }
+            }
```

Comment:
You should never have to do that. The approach here is to change any hardcoded HEX values within the SVG to use [`currentColor`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword). And to set the `color` property on `mx_Icon`.

Reply:
Right, I agree with you.

For the record: it is documented here → https://github.com/matrix-org/matrix-react-sdk/blob/74d30187a46b288aefcc746a5c83645f537f551d/docs/icons.md?plain=1#L11

Reply:
With https://github.com/matrix-org/matrix-react-sdk/pull/10955/commits/0b8671bb7aab57e5fd5fca115aad491ed7fe5570, the buttons are rendered as below.
![1](https://github.com/matrix-org/matrix-react-sdk/assets/3362943/3507bd9b-ebd8-4eec-9fcb-c1d21537e591)